Patches/Lemolim Race/Lemonim_Pawnkind.xml

```diff
@@ -0,0 +1,251 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+	<mods>
+	<li>Lemolim Race</li>
+	</mods>
+		<match Class="PatchOperationSequence">
+		 <operations>
+		 <!-- Tribal Pawns-->
+			<li Class="PatchOperationAdd">
+			<xpath>Defs/PawnKindDef[ @Name="LemolimTribalBase" or defName="LemolimTribal"]</xpath>
+				<value>
+				<apparelRequired>
+				<li>Apparel_TribalBackpack</li>
+				</apparelRequired>
+				</value>
+			</li>
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/PawnKindDef[ @Name="LemolimTribalBase"]/invNutrition</xpath>
+				<value>
+					 <invNutrition>1.8</invNutrition>
+				</value>
+				</li>
+			<li Class="PatchOperationAddModExtension">
+			<xpath>Defs/PawnKindDef[ defName="LemolimTribal_Archer" or defName="LemolimTribal_Hunter" or defName="LemolimTribal_HeavyArcher" or defName="LemolimTribal_ChiefRanged"]</xpath>
+			<value>
+			<li Class="CombatExtended.LoadoutPropertiesExtension">
+				<primaryMagazineCount>
+				<min>15</min>
+				<max>30</max>
+				</primaryMagazineCount>
+				<forcedSidearm>
+					<sidearmMoney>
+					<min>500</min>
+					<max>700</max>
+					</sidearmMoney>
+				<weaponTags>
+				<li>MedievalMeleeAdvanced</li>

```

Comment:
This should probably be a Neolithic tag, given the low tech level of the rest of their gear.